velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -497,10 +500,10 @@ bool registerMinMaxAggregate(const std::string& name) {
       });
 }
 
-static bool FB_ANONYMOUS_VARIABLE(g_AggregateFunction) =
-    registerMinMaxAggregate<MinAggregate, NonNumericMinAggregate>(kMin);
-static bool FB_ANONYMOUS_VARIABLE(g_AggregateFunction) =
-    registerMinMaxAggregate<MaxAggregate, NonNumericMaxAggregate>(kMax);
+void registerMinMaxAggregate() {
```

Comment:
As @mbasmanova suggested, I think this and some other registerXXAggregate() also need to be renamed as registerXXAggregates().